alerts/geomodel/config.py

```diff
@@ -0,0 +1,61 @@
+from typing import List, NamedTuple
+
+
+# TODO: Switch to dataclasses when we move to Python3.7+
+
+class Localities(NamedTuple):
+    '''Contains configuration for working with locality state.
+    '''
+
+    es_index: str
+    valid_duration_days: int
+    radius_kilometres: float
+
+
+class SearchWindow(NamedTuple):
```

Comment:
Is a class needed here if we only have one field? What if https://github.com/mozilla/MozDef/pull/1424/files#diff-691e6b4ee17eb7e5df421c87f045908cR28 was `search_window: int`